### PR TITLE
Main app

### PR DESCRIPTION
**Pull request by erezfree29**
This is a final capstone project app.
The app fetches information from the https://www.scorebat.com API.
It displays recent matches of Europe's top leagues, clicking on an image will link the user to a video of the match.

### CODE REVIEW

File: src/components/dr [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <div className="image">
+      <img src="https://a5.behance.net/11aa46df22ab4e63bdd2e24a88f638e6c74a8f7d/img/galleries/icons/ribbons/1x/graphic-design.png?cb=264615658" alt="gr" />
+    </div>
+    <div className="image">
+      <img className="image" src="https://a5.behance.net/11aa46df22ab4e63bdd2e24a88f638e6c74a8f7d/img/galleries/icons/ribbons/1x/ui-ux.png?cb=264615658" alt="ui" />
+      <div>2</div>
+    </div>
+  </div>
+</nav>
+</>
```
Comment (mricanho) [inline src/components/dr]: - [ ] This file doesn't belong to this project, please remove it

File: src/components/dra [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          <img className="ui" src="https://a5.behance.net/11aa46df22ab4e63bdd2e24a88f638e6c74a8f7d/img/galleries/icons/ribbons/1x/ui-ux.png?cb=264615658" alt="ui" />
+          <div className="two">2</div>
+          <div className="hide_two">
+            <div className="featured">FEATURED IN</div>
+            <div className="branding"><a href="https://www.behance.net/galleries/ui-ux/web-design" style={{ color: 'blue' }}>Web Design</a></div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </nav>
```
Comment (mricanho) [inline src/components/dra]: - [ ] This file doesn't belong to this project, please remove it

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @erezfree29, 

You are following the gitflow, your README file is complete, your commit messages are good and your video presentation was ok, you are already receiving information from the API, a good start! 🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] You have work to do, first read again the requirements for this capstone, this must be a single page application (SPA), it must have a home page and a details page, as well as a category filter in the home page, right now you only have a Home Page and when you click an element it points out of the application to www.scorebat.com, this is not the assignment, please add a details page with numeric information about the matches.

- [ ] You are not following the layout or the design, you need to copy the exact layout and choose a pallet color (whatever you like) that mimics the original design, this is a hard requirement.

imagen

Original design home page and details page:

imagen

Your design:

imagen

- [ ] Add more unit tests for pure functions in your Redux code.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**